cdap-docs/developers-manual/source/building-blocks/datasets/table.rst

```diff
@@ -92,6 +92,24 @@ the corresponding methods accept a default value to be returned when the column
   // Get column value as a primitive type or 0 if column is absent
   long valueAsLong = row.getLong("column1", 0);
 
+Multiple rows can be requested together using a variation of the ``get`` operation that takes a
+list of ``Get`` objects to be retrieved::
+
+  Table t;
+
+  // Define the rows to retrieve
+  List<Get> gets = Lists.newArrayList();
+  gets.add(new Get("row1"));
+  // Separate columns can be requested for each row
+  gets.add(new Get("row2").add("column1").add("column2"));
+  gets.add(new Get("row3"));
+
+  List<Row> rows = t.get(gets);
+
+Each ``Row`` object in the returned list will contain the results for one of the requested row
+keys.  When multiple rows must be retrieved together, this version of the ``get`` operation will
+allow the storage provider to do more efficient batching of the operations, if supported.
+
```

Comment:
We have minor mentions of it on the "Datasets" page of the Developers' Manual "Building Blocks" section; see http://stg-web101.sw.joyent.continuuity.net/cdap/2.8.0-SNAPSHOT/en/developers-manual/building-blocks/datasets/index.html. 
But we could add it in any of a number of places:
- we could add just a mention such as "the KeyValueTable uses this version of ``get`` in its implementation of ``readAll``.
- we could instead add it to the advanced topics page as a best practices

I just think that if we are going to mention this feature, but then give an example that uses it in an manner that hides the invocation, that we should explain that in some way to help people. (After spending a few days struggling with asynchronous Javascript calls that aren't well-documented, I'm in favor of writing more details...)